csrc/device_lower/pass/inline_ptx.cpp

```diff
@@ -264,25 +246,6 @@ class LowerToInlinePtx : public kir::ExprMutator {
                 /*volatile=*/true,
                 /*memory=*/false,
                 /*readable_outputs=*/{0}}));
-
```

Comment:
Sync insertion is now moved to indexing, still not good, but at least `LowerToInlinePtx` becomes cleaner.